components/fieldgroups/account/events/account-interesting-moment.schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/account/events/account-interesting-moment",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Account Interesting Moment",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "meta:conditionalField": "xdm:eventType",
```

Comment:
What does this "xdm:eventType" signify here? Should it be "xdm:accountEventType" instead?

Reply:
I adjusted this on all areas